config/template_test.go

```diff
@@ -27,7 +27,8 @@ func TestParseEnvironment(t *testing.T) {
 
 func TestTemplate(t *testing.T) {
 	env := parseEnvironment([]string{"NAME=Template", "USER=pilot", "PARTS=a:b:c"})
-
+	// To test env function
+	os.Setenv("NAME_1", "Template")
```

Comment:
Can you `defer` unsetting this variable after you set it here, so that we're not interfering with the global environment for the rest of the tests?